docs/configuration.md

```diff
@@ -99,6 +115,46 @@ You can also check for multiple values in a tag by separating the values by a co
 
 `highway->primary,secondary,tertiary`
 
+#### override
+
+This key allows you to specify certain values for your configuration for particular countries.
+The properties of this object must have keys that are equal to ISO3 country codes or group names defined in the groups object, as explained above.
+The values inside this object are themselves objects, which mimic the structure and properties of the default configuration for this check.
+Values defined inside of this country-specific override object take precedence over the default values, when run on a particular country.
+An example:
+```
+{
+  "CheckResourceLoader": {...},
+  "groups": {
+    "one": ["ABC", "DEF", "GHI"]
+  },
+  "CheckOne": {
+    "value": 1.0,
+    "word": "apple"
+    "override": {
+      "JKL": {
+        "value": 2.0,
+        "word": "banana"
+      },
+      "one": {
+        "value": 1.5,
+        "word": "orange"
+      }
+    }
+  }
+}
+```
+For the fictional country with ISO code `JKL`, `CheckOne` will be run with `value` equal to 2.0 and `word` equal to banana, while in `ABC`, `DEF`, and `GHI`, `value` will be 1.5 and `word` will be orange.
+In the rest of the world, the default values of 1.0 and apple will be used.
```

Comment:
Also, I believe something to note: If JKL were to be a part of group one. It's specific override would take precedent over the group override.